api_tests/preprints/views/test_preprint_files_list.py

```diff
@@ -360,7 +360,7 @@ def test_not_just_primary_file_returned(self):
 
         data = res.json['data']
         assert len(data) == 2
-        assert data[0]['id'] == self.preprint.primary_file._id
+        assert set([item['id'] for item in data]) == {second_file._id, self.preprint.primary_file._id}
```

Comment:
The new use of querysets over lists on [this line](https://github.com/CenterForOpenScience/osf.io/pull/9903/files#diff-c9e908790686537f7d767690c967d60794829495a57d838ff96d9685dd17c3deR659) makes the order ambiguous.